lib/mongo/database/view.rb

```diff
@@ -197,7 +200,16 @@ def initial_query_op(session, options = {})
       end
 
       def send_initial_query(server, session, options = {})
```

Comment:
I know this method predates your PR, but is it worth adding a comment to document it? For me, at least, it's not clear without a lot of digging what the method's intent is.

Reply:
Good hint, thank you! Added the comment.